core/templates/pages/admin-page/navbar/admin-navbar.directive.html

```diff
@@ -181,9 +180,8 @@
               </a>
               <ul uib-dropdown-menu class="dropdown-menu uib-dropdown-menu ng-cloak oppia-navbar-dropdown"
                   role="menu"
-                  ng-mouseover="$ctrl.onMouseoverProfilePictureOrDropdown($event)"
-                  ng-mouseleave="$ctrl.onMouseoutProfilePictureOrDropdown($event)"
-                  ng-show="$ctrl.profileDropdownIsActive">
+                  ng-mouseenter="$ctrl.profileDropdownIsActive = true"
+                  ng-mouseleave="$ctrl.profileDropdownIsActive = false">
```

Comment:
Why are we removing the ng-show check?

Reply:
https://drive.google.com/file/d/1Hv6rrIVuhgnoVityRdaXNeY3xOvXOW5Z/view?usp=sharing
Due to two checks i.e, ng-class and ng-show using the same variable as "$ctrl.profileDropdownIsActive" it is causing infinite loop and distortion, removing it solved the issue

Reply:
sorry I didn't get it, how ng-class and ng-iff creating an infinite loop?